File: app/src/main/java/com/example/ivan/foursquareapp/PlacesView.java
package com.example.ivan.foursquareapp;

import com.arellomobile.mvp.MvpView;
import com.example.ivan.foursquareapp.models.ItemVenue;

import java.util.List;


public interface PlacesView extends MvpView {

    void getPlaces(List<ItemVenue> list);

    void errorGetPlaces(String message);
}

File: app/src/main/java/com/example/ivan/foursquareapp/presentation/places/PlacesView.kt
package com.example.ivan.foursquareapp.presentation.places

import com.arellomobile.mvp.MvpView
import com.example.ivan.foursquareapp.models.ItemVenue


interface PlacesView : MvpView {

    fun getPlaces(list: List<ItemVenue>)

    fun errorGetPlaces(message: String)
}
